doczrc.js

```diff
@@ -1,6 +1,6 @@
 export default {
   indexHtml: './index.html',
-  menu: ['Home', 'Button', 'Input','Display Card', 'Dropdown', 'CheckboxGroup', 'RadioGroup', 'Form', 'Modal', 'Panel', 'Table', 'Tag'],
+  menu: ['Home', 'Button','CheckboxGroup','Display Card', 'Dropdown','Form', 'Input', 'Loading Skeleton', 'Modal', 'Panel','RadioGroup', 'Table', 'Tag'],
```

Comment:
Good call rearranging these-- maybe not a bad idea to break them into separate lines so it's easier to read through (also for Git). Somethin like this:
```
menu: [
    'Home', 
    'Button',
    'CheckboxGroup',
    'Display Card',
    'Dropdown',
    'Form',
    'Input',
    'Loading Skeleton',
    'Modal',
    'Panel',
    'RadioGroup',
    'Table',
    'Tag',
  ],
```

Reply:
Fixed menu to make it easier to read